docs/reference/commandline/run.md

```diff
@@ -256,365 +258,39 @@ explains in detail how to manipulate ports in Docker.
-
-The `my-label` key doesn't specify a value so the label defaults to an empty
-string(`""`). To add multiple labels, repeat the label flag (`-l` or `--label`).
-
-The `key=value` must be unique to avoid overwriting the label value. If you
-specify labels with identical keys but different values, each subsequent value
-overwrites the previous. Docker uses the last `key=value` you supply.
-
-Use the `--label-file` flag to load multiple labels from a file. Delimit each
-label in the file with an EOL mark. The example below loads labels from a
-labels file in the current directory:
-
-    $ docker run --label-file ./labels ubuntu bash
-
-The label-file format is similar to the format for loading environment
-variables. (Unlike environment variables, labels are not visible to processes
-running inside a container.) The following example illustrates a label-file
-format:
-
-    com.example.label1="a label"
-
-    # this is a comment
-    com.example.label2=another\ label
-    com.example.label3
-
-You can load multiple label-files by supplying multiple  `--label-file` flags.
-
-For additional information on working with labels, see [*Labels - custom
-metadata in Docker*](../../userguide/labels-custom-metadata.md) in the Docker User
-Guide.
-
-### Connect a container to a network (--network)
-
-When you start a container use the `--network` flag to connect it to a network.
-This adds the `busybox` container to the `my-net` network.
 
-```bash
-$ docker run -itd --network=my-net busybox
-```
+#### Set metadata on container (-l, --label, --label-file)
 
-You can also choose the IP addresses for the container with `--ip` and `--ip6`
-flags when you start the container on a user-defined network.
+For information on working with labels, see [*Labels - custom
+metadata in Docker*](../../userguide/labels-custom-metadata.md) in the Docker
+User Guide.
 
-```bash
-$ docker run -itd --network=my-net --ip=10.10.9.75 busybox
-```
+#### Connect a container to a network (--network)
 
-If you want to add a running container to a network use the `docker network connect` subcommand.
+To connect a container to a network managed by Docker, use the `--network` flag.
+This example adds the `busybox` container to the `my-net` network.
 
-You can connect multiple containers to the same network. Once connected, the
-containers can communicate easily need only another container's IP address
-or name. For `overlay` networks or custom plugins that support multi-host
-connectivity, containers connected to the same multi-host network but launched
-from different Engines can also communicate in this way.
+```bash
+$ docker run -itd --network="my-net" busybox
+```
 
-**Note**: Service discovery is unavailable on the default bridge network.
-Containers can communicate via their IP addresses by default. To communicate
-by name, they must be linked.
+See the [Docker run reference](../run.md) for more details about configuring
+network settings at runtime.
```

Comment:
This section may be more relevant https://docs.docker.com/engine/userguide/networking/